collections/_posts/2022-11-28-CurriculumLearningByDIH.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: review
+title: "Curriculum learning by Dynamic Instantaneous Hardness"
+tags: deep-learning classification curriculum-learning
+author: "Juliette Moreau"
+cite:
+    authors: "Tianyi Zhou, Shengjie Wang, Jeff A. Bilmes"
+    title:   "Curriculum learning by Dynamic Instantaneous Hardness"
+    venue:   "NeurIPS 2020"
+pdf: "https://dl.acm.org/doi/pdf/10.5555/3495724.3496445"
+---
+
+# Notes
+
+* Code is available [here](https://github.com/tianyizhou/DIHCL)
+
+# Highlights
+
+* This paper proposes a new difficulty metric for curriculum learning called Dynamic Instantaneous Hardness (DIH).
+* Its purpose is to speed-up learning and to improve the results.
+* It is only tested on classification but might be extended to other applications.
+&nbsp;
+
+# Introduction
+
+Curriculum learning[^1] consists in presenting the right examples in the right order at the right time during training to enhance the learning process, just as teachers would do with their students. To that end, it is necessary to define a concept of _hardness_. It can be related to the different classes -in the case of classification- or inherent to the examples, such as specific shapes. One approach, called self-paced learning[^2], selects some examples at each epoch based on instantaneous feedback from the model. However, it does not take into account the training history of each sample, motivating the introduction of DIH.
+
+# Methods
+
+## Dynamic Instantaneous Hardness
+
+The goal of the measure is to get the usefulness of each sample in the future in order to select them wisely. The Dynamic Instantaneous Hardness (DIH) is defined as the exponential moving average of instantaneous hardness measures of a sample over time, the instantaneous hardness $$a_t(i)$$ being a measure retrieved from the epoch that has just been computed.
+
+$$
+r_{t+1}(i) = \left\{
+  \begin{array}
+    1 \gamma*a_t(i) + (1-\gamma)*r_{t}(i) \quad if \quad i \in S_t \\
+    r_t(i) \quad else
+  \end{array}
+  \right.
+$$
+
+$$ \gamma \in [0,1] $$ is a discount factor and $$S_t$$ is the set of samples used for training at time $$t$$.
+
+The three instantaneous hardness used to calculate DIH and compared are:
```

Comment:
Now that what you mean is clear, I'd suggest to reformulate it more succintly as:
```suggestion
Three definitions of instantaneous hardness are compared:
```